### PR TITLE
Another batch of updates to our Jenkins docs.

### PR DESCRIPTION
**Pull request by bobsummerwill**

### CODE REVIEW

File: source/ethereum-clients/cpp-ethereum/automation.rst [hunk 1]
```diff
@@ -87,3 +84,51 @@ some key scripts which our automation uses:
...
+    - brew update
+    - brew upgrade
+    - install pre-requisites (http://www.ethdocs.org/en/latest/ethereum-clients/cpp-ethereum/building-from-source/osx.html)
+    - install Ruby
+        - See https://github.com/rbenv/rbenv#homebrew-on-mac-os-x
+        - brew install rbenv
+        - rbenv init
+        - rbenv install 1.9.3-p551
+        - Add `eval "$(rbenv init -)"` to ~/.bash_profile:
+    - Connect the slave using Java web-start (have to lower security settings)
```
Comment (chriseth) [inline source/ethereum-clients/cpp-ethereum/automation.rst]: That could be changed to a setting where the server logs in via ssh, by the way.

File: source/ethereum-clients/cpp-ethereum/automation.rst [hunk 2]
```diff
@@ -87,3 +84,51 @@ some key scripts which our automation uses:
...
+-------------------------------------------------------------------------------
+
+- Lack of canonical build
+- Lack of webthree-umbrella builds
+- No automated Windows tests
+- Broken cross-repo PRs
+- Hanging tests
+- Incomplete test suits
+- Do we run "ethereum/tests" at all?
+- Do we run "ethereum/rpc-tests" at all?
```
Comment (chriseth) [inline source/ethereum-clients/cpp-ethereum/automation.rst]: No, they have been non-functional for a long time.

File: source/ethereum-clients/cpp-ethereum/automation.rst [hunk 3]
```diff
@@ -87,3 +84,51 @@ some key scripts which our automation uses:
...
+Known issues
+-------------------------------------------------------------------------------
+
+- Lack of canonical build
+- Lack of webthree-umbrella builds
+- No automated Windows tests
+- Broken cross-repo PRs
+- Hanging tests
+- Incomplete test suits
+- Do we run "ethereum/tests" at all?
```
Comment (chriseth) [inline source/ethereum-clients/cpp-ethereum/automation.rst]: Yes, from ethtest within libethereum.

File: source/ethereum-clients/cpp-ethereum/automation.rst (hunk 1 above)
Comment (bobsummerwill) [inline source/ethereum-clients/cpp-ethereum/automation.rst]: What specifically could be changes to a login via SSH?

**Pull request merged**